.travis.yml

```diff
@@ -10,8 +10,8 @@ env:
   global:
     - COMPOSER_MEMORY_LIMIT=2G
   matrix:
-    - TEST_SUITE=8.6.x
```

Comment:
I guess this is still OK? Or maybe it came in from a merge, but is still showing up?

Reply:
This is ok. See https://github.com/Gizra/og/pull/473